src/vehicle.h

```diff
@@ -1695,6 +1703,7 @@ class vehicle
 
     public:
         bool is_autodriving = false;
+        bool is_following = false;
```

Comment:
This is not stored in the save. See savegame_json.cpp for storing and loading it.

Reply:
oops yeah.